ebusd-2.1.x/de/vaillant/08.hmu.csv

```diff
@@ -1,6 +1,37 @@
 # type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment,field2,part (m/s),datatypes/templates,divider/values,unit,comment,field3,part (m/s),datatypes/templates,divider/values,unit,comment,field4,part (m/s),datatypes/templates,divider/values,unit,comment
+*r,,,,,,"B507",09,,,,,,
+#r,,Unknown,,,,,,,m,temp1,,,
+*w,,,,,,"B507",09,,,,,,
+#w,,Unknown,,,,,,,m,temp1,,,
```

Comment:
unused, please remove (same in "en" file)

Reply:
Hi John,

Here are some examples of the output (I removed the status translation from the ebusd configuration files in my setup as I have a multiple language user interface)

In 08.hmu.csv

VWZ Live monitor values:

Main status:
[update info] sent MS cmd: 3108b503020003 / 0af401ffffffffffffffff
[update notice] sent read hmu LiveMonitorMain: 244;Heating

Modulation. Datatype is D1B
[update info] sent MS cmd: 3108b51a0405003225 / 0a00081b00000000000000
[update notice] sent read hmu LiveMonitor QQ=31: 25;00 00

Power consumption. Datatype is UIN
[update info] sent MS cmd: 3108b51a0405003224 / 0a00083500000000000000
[update notice] sent read hmu LiveMonitor QQ=31: 24;00 00

Air intake temperature. Datatype is D2C
[update info] sent MS cmd: 3108b51a0405003226 / 0a000808df000000000000
[update notice] sent read hmu LiveMonitor QQ=31: 26;df 00

Desired supply temperature. Datatype is D2C
[update info] sent MS cmd: 3108b51a040500321f / 0a000805f0000000000000
[update notice] sent read hmu LiveMonitor QQ=31: 1f;f0 00
 
Actual supply temperature. Datatype is D2C
[update info] sent MS cmd: 3108b51a0405003220 / 0a0008081b010000000000
[update notice] sent read hmu LiveMonitor QQ=31: 20;1b 01

Power generated. Datatype is UIN
[update info] sent MS cmd: 3108b51a0405003223 / 0a00083500000000000000
[update notice] sent read hmu LiveMonitor QQ=31: 23;00 00

VWZ test menu’s:

ReadEEVPosition
[update info] sent MS cmd: 3108b51405051503ffff / 0415002600
[update notice] sent read hmu EEVPosition QQ=31: 38

ReadCompressorIntakeTemp
[update info] sent MS cmd: 3108b51405053803ffff / 0438008300
[update notice] sent read hmu CompressorIntakeTemp QQ=31: 13.1

Additional thermostat items (tested for 15.700.csv and 15.b7v00.csv (wireless 700)):
[update info] sent MS cmd: 3115b524080201000006000000 / 020000
[update notice] sent write b7v00 ManualCooling QQ=31: 0

There are many more items but they all work similar to the menu items above

Let me know if you need more information
Eric